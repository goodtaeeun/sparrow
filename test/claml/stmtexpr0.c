int main() {
  int x = ({
    int d = 1;
    d;
  });
  return 0;
}
