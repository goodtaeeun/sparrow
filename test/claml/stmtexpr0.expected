/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "stmtexpr0.c"
int main(void) {
  int x;

  {
#line 2
    x = 0;
#line 6
    return (0);
  }
}
